### PR TITLE
🤖📯 GitHub OpenAPI Update

### PR DESCRIPTION
**Pull request by octokitbot**
I found new changes in [GitHub's official OpenAPI specifications](https://github.com/github/rest-api-description) and thought I'd let you know about it 👋🤖

I can't tell if the changes include fixes, features, breaking changes or just cache updates, you'll have to figure that out on yourself and adapt the commit messages accordingly to trigger the right release, see [our commit message conventions](https://github.com/octokit/openapi/blob/main/CONTRIBUTING.md#merging-the-pull-request--releasing-a-new-version).

### CODE REVIEW

File: generated/github.ae-anicca-diff-to-api.github.com.deref.json [hunk 1]
```diff
@@ -10604,31 +10608,9 @@
       }
     },
     "removed": {
```
Comment (nickfloyd) [inline generated/github.ae-anicca-diff-to-api.github.com.deref.json:10610]: It appears that a good handful of authorizations / apps endpoints and operations were deprecated.
This looks to be a breaking change.

**Review by gr2m — COMMENTED** [review]
I looked at the changes in `` and they look normal. 

Probably all the additions come from adding these new files:

```
cache/ghes-3.6.deref.json
cache/ghes-3.6.json
generated/ghes-3.5-anicca-diff-to-ghes-3.6.deref.json
generated/ghes-3.5-diff-to-ghes-3.6.deref.json
generated/ghes-3.5-diff-to-ghes-3.6.json
generated/ghes-3.6-anicca-diff-to-api.github.com.deref.json
generated/ghes-3.6-diff-to-api.github.com.deref.json
generated/ghes-3.6-diff-to-api.github.com.json
generated/ghes-3.6.deref.json
generated/ghes-3.6.json
```

Regarding the removals: they have been long overdue. I would say we could release a breaking change for `octokit/openapi` and the `@octokit/openapi-types` package, but it's not a breaking change for the libraries.

However, from past experiences, remove these nodes cause weird type problems because types that previously existed no longer do. My hope is we get around that be releasing the breaking version of `@octokit/openapi-types`, so folks won't just get it with a fresh install, we will need to release a dedicated fix version for the `@octokit/*` packages for the new types to be used.

Changes to `x-github.{category,subcategory}` have no effect on user-facing API as far as I know, I think we use it for the docs at https://octokit.github.io/rest.js (which really could use some love, but that's its own project). The code only uses the operation ID to generate method names

**Comment by timrogers** [comment]
@nickfloyd I've gone through the changes in the non-dereferenced file for `api.github.com` and created commits for each change, which you can now read through and review. (I'm making the (questionable) assumption that there won't be anything in the other versions.)

There aren't any real breaking changes in here, in the sense that anything that is breaking from a schema perspective is just reflecting how the API has always been (or at least how it has been since 2020!).

**Review by nickfloyd — DISMISSED** [review]
👍

**Comment by timrogers** [comment]
Merging, as this was previously reviewed by @nickfloyd

**Pull request merged**